modules/claims_api/app/sidekiq/claims_api/ews_updater.rb

```diff
@@ -29,38 +29,6 @@ def perform(ews_id)
 
     private
 
-    def update_claim_level_suspense(ews)
-      suspense_claim = claim_management_service(ews).find_claim_level_suspense(claim_id: ews.claim_id)
-      updated_claim = update_suspense_date(claim: suspense_claim)
-      omitted_claim = omit_fields(updated_claim)
-      claim_management_service(ews).update_claim_level_suspense(claim: omitted_claim)
-      success_message = "Successfully updated suspense dates for claim #{ews.claim_id} "
-      ClaimsApi::Logger.log('ews_updater', ews_id: ews.id, detail: success_message)
-      ews.bgs_error_message = nil if ews.bgs_error_message.present?
-      ClaimsApi::EvidenceWaiverSubmission::UPDATED
-    rescue => e
```

Comment:
@stiehlrod  I am wondering if we still need a rescue somewhere just in case an error does get raised from something.  Looks like this would have caught any issues that came up from any of the 'service' calls, Do we need a rescue maybe just moved into the `perform` just to make sure anything gets caught.  Again, just let me know.